tendrl/node_agent/objects/cpu/__init__.py

```diff
@@ -0,0 +1,28 @@
+from tendrl.commons.etcdobj import EtcdObj
+from tendrl.node_agent.manager.pull_hardware_inventory import getNodeCpu
+from tendrl.node_agent import objects
+
+
+class Cpu(objects.NodeAgentBaseObject):
+    def __init__(self, architecture=None, cores_per_socket=None,
+                 cpu_family=None, cpu_op_mode=None,
+                 model=None, model_name=None, vendor_id=None,
+                 *args, **kwargs):
+        super(Cpu, self).__init__(*args, **kwargs)
+        cpu = getNodeCpu()
+        self.value = 'nodes/%s/Cpu'
+        self.architecture = architecture or cpu['Architecture']
+        self.cores_per_socket = cores_per_socket or cpu["CoresPerSocket"]
+        self.cpu_family = cpu_family or cpu["CPUFamily"]
+        self.cpu_op_mode = cpu_op_mode or cpu["CpuOpMode"]
+        self.model = model or cpu["Model"]
+        self.model_name = model_name or cpu["ModelName"]
+        self.vendor_id = vendor_id or cpu["VendroId"]
+        self.etcd_cls = _CpuEtcd
```

Comment:
make it self._etcd_cls  (this attr needs to be private otherwise it gets saved to etcd)